api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/ldap/AbstractLdapProperties.java

```diff
@@ -41,6 +41,22 @@
      * by the underlying Java platform.
      */
     private String keystoreType;
+
+    /**
+     * Name of the truststore to use for the SSL or startTLS connection.
+     */
+    private String trustStore;
```

Comment:
Name or path? Name reads confusing. Does this mean I can name my truststore anything?